### PR TITLE
Adjust tests for Kuberay 1.0

### PR DESCRIPTION
**Pull request by jiripetrlik**
Tests on master branch are for Kuberay 1.1.

### CODE REVIEW

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-kuberay-e2e.robot [hunk 1]
```diff
@@ -9,7 +9,7 @@ Resource          ../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
 *** Variables ***
 ${KUBERAY_DIR}            kuberay
 ${KUBERAY_REPO_URL}       %{KUBERAY_REPO_URL=https://github.com/ray-project/kuberay.git}
```
Comment (sutaakar) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-kuberay-e2e.robot:11]: Can you rather use https://github.com/red-hat-data-services/kuberay as this is the source code for product?

**Review by jgarciao — DISMISSED** [review]

**Comment by sutaakar** [comment]
Liine number 49 is referencing wrong variables:
`FAIL    Unable to clone Kuberay repo ${DW_REPO_URL}:${DW_REPO_BRANCH}:${DW_DIR}`

**Comment by sutaakar** [comment]
Can you add a waiting command after enabling Ray component, waiting for Ray to be up and running (or maybe ready in DSC status)?
Locally when I ran the test it fails, seems like the test already expects Ray to be running, while it takes time for RHOAI to spin up Ray after t is enabled.

**Comment by sutaakar** [comment]
Can you also pass custom Ray image as env variable for test? See https://github.com/red-hat-data-services/kuberay/blob/master/ray-operator/test/support/environment.go#L15 .
Default image refer to dockerhub, in our tests we can reference i.e. https://github.com/project-codeflare/codeflare-common/blob/main/support/defaults.go#L10

**Comment by jiripetrlik** [comment]
@sutaakar Thank you for your comments. It should be all fixed now. I was able to successfully run it also on Jenkins.

**Review by sutaakar — APPROVED** [review]
Looks good to me, thanks

**Review by jgarciao — APPROVED** [review]

**Pull request merged**